lib/hyrax/configuration.rb

```diff
@@ -373,6 +373,11 @@ def subject_prefix
       @subject_prefix ||= "Contact form:"
     end
 
+    attr_writer :extract_full_text
+    def extract_full_text?
+      @extract_full_text ||= false
```

Comment:
Yeah - that's what I thought. I was having issues testing until this setting. I'll work it the other way and see what was up with my test.